app/views/step_by_step_pages/_form.html.erb

```diff
@@ -31,7 +31,7 @@
   </div>
 
   <div class="form-group">
-    <%= form.submit "Save and continue", class: "btn btn-primary" %>
+    <%= form.submit "Save", class: "btn btn-primary" %>
```

Comment:
This should be moved into a separate commit as renaming the button is related to the change of navigation styles.

Reply:
Ref: https://github.com/alphagov/collections-publisher/pull/438/commits/cb3c7538bc7512aa3b3ec69bcc65f5babffbce07